vkconfig/vkconfig.md

```diff
@@ -149,6 +149,8 @@ We are working on defining layers development conventions to resolve this issue
 --------------
 ## Known Issues
 
+- The UI still feels a little clunky... Need more polish.
```

Comment:
This phrase in really bad I think in English. Clunky is a very derogatory and negative term, and you poison peoples expectations by using it. Don't tell them right away the user interface is horrible... tell them there are some issues we still want to clean up, and let them tell us what/if they find anything objectionable.

Reply:
"UI still needs some refinement in some areas"
A much better spin...